design/frontend/base/default/template/bold/checkout_payment_booster/payment/form/base.phtml

```diff
@@ -0,0 +1,672 @@
+<?php
+/**
+ * @var Bold_CheckoutPaymentBooster_Block_Payment_Form_Base $this
+ */
+?>
+
+<script type="text/javascript">
```

Comment:
Rather than putting this JavaScript into a template, we should put it in a proper JavaScript file and pass in the parameters as necessary.

Reply:
In M1 to pass all necessary params like:
        isAvailable: <?php echo $this->isAvailable(); ?>,
        groupLabel: '<?php echo $this->getGroupLabel() ?>',
        boldApiUrl: '<?php echo $this->getBoldApiUrl(); ?>',
        jwtToken: '<?php echo $this->getJwtToken(); ?>',
        epsAuthToken: '<?php echo $this->getEpsAuthToken(); ?>',
        epsUrl: '<?php echo $this->getEpsUrl(); ?>',
        epsStaticUrl: '<?php echo $this->getEpsStaticUrl(); ?>',
        epsGatewayId: '<?php echo $this->getEpsGatewayId(); ?>',
        quoteCurrencyCode: '<?php echo $this->getQuoteCurrencyCode(); ?>',
        isFastlaneAvailable: <?php echo $this->isFastlaneAvailable(); ?>,
we need to introduce template. So, getting javascript out of temple into separate js file will complicate onboarding process and we still need to add additional block and template to later load and pass these params in to js file.

Reply:
That is the way that both Magento 1 and Magento 2 core does things. I'm using that methodology for the implementation of Express Pay right now.